README.md

```diff
@@ -21,8 +21,8 @@ designed for efficient, high-accuracy simulations of a variety of
 molecular properties. We can routinely perform computations with more
 than 2500 basis functions running serially or on multi-core machines.
 
-With computationally demanding portions written in C++, Pybind11 exports
-many of the C++ classes into Python, and a flexible Python driver, Psi4
+With computationally demanding portions written in C++, exports
+many C++ classes into Python via Pybind11, and a flexible Python driver, Psi4
```

Comment:
I am not a native speaker, but wouldn't: 
`With computationally demanding portions written in C++, exports *of* many C++ classes into Python ...`
read better?

Reply:
definitely. I was mainly avoiding the misreading of "pybind11 has its computationally demanding portions written in C++" and missed the preposition. thanks.